export.go

```diff
@@ -34,6 +35,15 @@ type Exporter struct {
 
 // NewExporter creates a new Exporter. Callers must call Close() when done.
 func newExporter(tree *ImmutableTree) *Exporter {
+	if tree == nil {
+		return nil
+	}
+	// CV Prevent crash on incrVersionReaders if tree.ndb == nil
+	if tree.ndb == nil {
+		fmt.Printf("iavl/export newExporter failed to create because tree.ndb is nil\n")
```

Comment:
this function should return error. Doing `fmt.Printf` doesn't make sense here.

Reply:
agree, lets break the api.

Reply:
I can just return (nil, err) with fmt.Errorf to start with.  Then any more complicated error handling can be followed up in other commits/improvment?

Or is there some error standard you're trying to maintain?

Reply:
yea that makes sense, we would then handle this in the application

Reply:
Returning error now, updated api/test